vendor/github.com/hashicorp/go-azure-helpers/authentication/config.go

```diff
@@ -14,6 +14,7 @@ import (
 // new Azure management client.
 type Config struct {
 	ClientID           string
+	ClientSecret       string
```

Comment:
This diff is definitively not intended to be here.

Reply:
Thanks @jrauschenbusch - I'll fetch your branch and figure out what's going on, we had to make an emergency fix to the master branch on Monday which is likely to be the root of the issue.

Reply:
I _think_ it's sorted now, will let Travis do it's thing and loop back shortly, we're a bit stretched today.